src/org/traccar/MainEventHandler.java

```diff
@@ -71,6 +71,16 @@ public void channelRead(ChannelHandlerContext ctx, Object msg) {
             if (position.getAccuracy() > 0) {
                 s.append(", accuracy: ").append(String.format("%.1f", position.getAccuracy()));
             }
+            if (position.getOutdated()) {
```

Comment:
I would prefer to avoid adding more logging.

Reply:
currently there are updates (many, for example in my case every minute) that don't have position but do have steps and battery. Currently it's displaying outdated position for every update like that. Basically the best would be in my opinion to display the data related to the actual update. If battery was updated, then display it, but not the other things that were not updated. If position was updated then display it. etc...

So what about the following change:

- don't display the location if it's outdated (and no need to display "outdated")

- I would like to display battery, and maybe other things. Actually maybe it would be worth to make it configurable. You will set it to position only, and I will set it to position,battery,steps and anyone else can chose for themselves. What do you think?

Reply:
made it more general: 1. it's displayed in the same order as it appears in the config, 2. you can choose any attribute